backend-service/app/utils/KafkaConfig.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package utils;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+
+/**
+ * Utilities for Kafka configurations and topics
+ */
+public class KafkaConfig {
+
+  private static Properties _props = new Properties();
+  // Map of <topic_name, topic_content>
+  private static Map<String, Topic> _topics = new HashMap<>();
+
+  public static final String GET_KAFKA_PROPERTIES = "SELECT * FROM wh_kafka_property ORDER BY `wh_kafka_property_id`";
+  public static final String GET_KAFKA_TOPICS = "SELECT * FROM wh_kafka_topic";
+
+  /**
+   * Class for storing Kafka Topic info
+   */
+  public static class Topic {
+    public String topic;
+    public int numOfWorkers; // number of kafka consumer workers
+    public String processor; // processor class to invoke
+    public String dbTable; // Database table to write to
+
+    Topic(String topic, int numOfWorkers, String processor, String dbTable) {
+      this.topic = topic;
+      this.numOfWorkers = numOfWorkers;
+      this.processor = processor;
+      this.dbTable = dbTable;
+    }
+  }
+
+  /**
+   * get Kafka properties from database, also update _props
+   * @return Properties
+   * @throws Exception
+   */
+  public static Properties getKafkaProperties() throws Exception {
+    final List<Map<String, Object>> rows = JdbcUtil.wherehowsJdbcTemplate.queryForList(GET_KAFKA_PROPERTIES);
```

Comment:
Let's use wh_etl_job_property to store the Kafka configuration

Multi-value properties can be expressed as comma-separated list.